coresystems.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
After talking with @kode-monki, I suspect this entire "Core Systems" feature will be removed from the IDP Profile Manager. It was taking the idea of a single starting point for accessing all the core systems (aka a "launchpad") and combining it with the IDP Profile Manager, but after discussion it seems like the launch pad needs to be a separate system, which may very likely link to the IDP Profile manager, but which will not be part of the IDP Profile Manager.

Since that will likely affect several aspects of this feature branch, would you prefer I continue reviewing this PR as-is, or give you time to make those changes before I review this further?

Reply:
You can give me more time. It will probably need some design anyways. Thanks for the clarity.